Apps/WebClient/src/ClientApp/src/store/modules/immunization/util.ts

```diff
@@ -0,0 +1,41 @@
+import Vue from "vue";
+
+import { ImmunizationDatasetState } from "@/models/datasetState";
+import { LoadStatus } from "@/models/storeOperations";
+
+import { ImmunizationState } from "./types";
+
+export const defaultImmunizationDatasetState: ImmunizationDatasetState = {
+    data: [],
+    status: LoadStatus.NONE,
+    statusMessage: "",
+    error: undefined,
+    recommendations: [],
+};
+
+/**
+ * Retrieves the immunization dataset state for a particular HDID.
+ * @param state The store state.
+ * @param hdid The HDID associated with the dataset state.
+ * @returns The dataset state for the HDID, if it exists, or a new dataset state initialized to default values.
+ */
+export function getImmunizationDatasetState(
+    state: ImmunizationState,
+    hdid: string
+): ImmunizationDatasetState {
+    return state.immunizations[hdid] ?? { ...defaultImmunizationDatasetState };
+}
+
+/**
+ * Updates the health visit state for a particular HDID.
```

Comment:
Nit: "health visit" should be "immunization".

Reply:
Fixed